example/components/Button.js

```diff
@@ -7,6 +7,7 @@ function Button(props) {
       onPress={props.onPress}
       style={props.style.container}
       nativeID={props.nativeID}
+      testID={props.testID}
```

Comment:
this, and the change in `TextInput` below are simply to support using `testID` properties in place of `nativeID` properties in the example app testing, if desired. At this point, none of the button or text input usages are doing this though, only the new tab bar usage in `MainScreen` via `tabBarTestID`.